### PR TITLE
Tests Components

### PR DESCRIPTION
**Pull request by touredri**
In this PR we have the following:

- [ ] Jest unit test
- [ ] components test with react testing library

### CODE REVIEW

File: src/tests/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    const obj2 = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    const answer = calculate(obj, 'AC');
+    expect(JSON.stringify(answer)).toEqual(JSON.stringify(obj2));
+    expect(calculate({ next: '5' }, 'AC')).toHaveProperty('total', null);
+  });
+});
```
Comment (anagudelogu) [inline src/tests/calculate.test.js]: - [x] You did a good job working on this milestone! 💯 However, according to the requirements:

image

Please make sure to test all functionalities and scenarios of the `calculate.js` file, at the moment you are just testing one! 🤓 🔧

File: src/tests/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+describe('operate', () => {
+  it('adds two numbers', () => {
+    expect(operate(2, 3, '+')).toBe('5');
+  });
+
+  it('multiple two numbers', () => {
+    expect(operate(2, 3, 'x')).toBe('6');
+  });
+});
```
Comment (anagudelogu) [inline src/tests/operate.test.js]: - [x] Here happens the same, please make sure to test all functionalities and scenarios for the `operate.js` file! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @touredri @Ouail-01, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/calculate.test.js (hunk 1 above)
Comment (rloterh) [reply src/tests/calculate.test.js]: well resolved! ✔️

File: src/tests/operate.test.js (hunk 2 above)
Comment (rloterh) [reply src/tests/operate.test.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ouail & Drissa,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job implementing test cases for the calculator logic aspect!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure that the _Getting Started_ section_ is filled with the appropriate instructions on how to _run test cases_ your app.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**